source/CAS/PGTStorage/Memcached.php

```diff
@@ -0,0 +1,31 @@
+<?php
+class CAS_PGTStorage_Memcached extends CAS_PGTStorage_AbstractStorage
+{
+    var $_cache;
+
+    function __construct($servers, $ncopies = 2, $expiration = 30){
```

Comment:
Can you please elaborate on the purpose of the `$ncopies` parameter? Does the memcached client not have failover support?